wai-extra/Network/Wai/Header.hs

```diff
@@ -2,6 +2,7 @@
 -- | Some helpers for dealing with WAI 'Header's.
 module Network.Wai.Header
     ( contentLength
+    , dropWhileEnd
```

Comment:
This seems more reasonable for a shared, unexposed utils module.

Reply:
Done :)